com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [11.0.x] - 202x-xx-xx
 ### Added
 - Added real-time Point Light Shadows.
+- Added support for custom additional (=punctual) light shadow resolutions.
```

Comment:
Is this a typo? (=

Reply:
Thank for pointing out that it is not clear ; I meant that additional are punctual lights.
I updated the note to make it more clear in db8cef6.

Reply:
the = sign is it supposed to be there?

Reply:
Saw the fix after posted. Thanks looks goo now. You can resolve this